specification/compute/resource-manager/Microsoft.Compute/2017-12-01/compute.json

```diff
@@ -5897,6 +5951,18 @@
         "licenseType": {
           "type": "string",
           "description": "Specifies that the image or disk that is being used was licensed on-premises. This element is only used for images that contain the Windows Server operating system. <br><br> Possible values are: <br><br> Windows_Client <br><br> Windows_Server <br><br> If this element is included in a request for an update, the value must match the initial value. This value cannot be updated. <br><br> For more information, see [Azure Hybrid Use Benefit for Windows Server](https://docs.microsoft.com/azure/virtual-machines/virtual-machines-windows-hybrid-use-benefit-licensing?toc=%2fazure%2fvirtual-machines%2fwindows%2ftoc.json) <br><br> Minimum api-version: 2015-06-15"
+        },
+        "priority": {
```

Comment:
Addition of settable properties to an existing model (`VirtualMachineScaleSetVMProfile`) of an existing api version is considered as breaking change. 

It looks like this model `VirtualMachineScaleSetVMProfile` is used in `VirtualMachineScaleSet` model [VirtualMachineScaleSet -> VirtualMachineScaleSetProperties -> VirtualMachineScaleSetVMProfile] which represent payload of already supported VMSS create or update op. that means we are adding settable properties to VMSS create_update payload.

Adding ARM to review this.